eternalcore-core/src/main/java/com/eternalcode/core/feature/fun/endscreen/EndScreenCommand.java

```diff
@@ -0,0 +1,48 @@
+package com.eternalcode.core.feature.fun.endscreen;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.notice.NoticeService;
+import com.eternalcode.paper.PaperOverlay;
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+import org.bukkit.entity.Player;
+
+@Command(name = "endscreen", aliases = {"end-screen", "win-screen"})
+@Permission("eternalcore.fun.endscreen")
+public class EndScreenCommand {
```

Comment:
This command should be annotated wih `@Compatibility` annotation. 

#showWinScreen has been introduced in 1.19.4 paper builds.

<img width="1909" height="175" alt="image" src="https://github.com/user-attachments/assets/3fac267a-8f10-4ad5-8ece-3b5776673747" />
<img width="1903" height="223" alt="image" src="https://github.com/user-attachments/assets/577a1fc0-a604-4d38-acbb-8de159e4a27b" />